pkgs/development/libraries/libopenraw/default.nix

```
@@ -0,0 +1,29 @@
+{ stdenv, fetchurl, boost, gdk-pixbuf, glib, libjpeg, libxml2, lib, pkg-config, cargo, rustc }:
+
+stdenv.mkDerivation rec {
+  pname = "libopenraw";
+  version = "0.3.7";
+
+  src = fetchurl {
+    url = "https://libopenraw.freedesktop.org/download/${pname}-${version}.tar.bz2";
+    sha256 = "5515b2610361ef34580b6b976635119f6dedb4f0a79d54662fa5fe6186a45ed5";
```

`nix-hash --to-sri --type sha256 5515b2610361ef34580b6b976635119f6dedb4f0a79d54662fa5fe6186a45ed5` or setting `hash = "";` will get you the hash below:

```suggestion
    hash = "sha256-VRWyYQNh7zRYC2uXZjURn23ttPCnnVRmL6X+YYakXtU=";
```